R/fmelt.R

```diff
@@ -19,14 +19,14 @@ melt.default = function(data, ..., na.rm = FALSE, value.name = "value") {
   # nocov end
 }
 
-patterns = function(..., cols=character(0L)) {
+patterns = function(..., cols=character(0L), perl=FALSE) {
```

Comment:
why not include the rest of the `grep()` signature while we're at it, namely `ignore.case`, `fixed`, `useBytes` could be useful. `value` and `invert` don't seem so useful.

Reply:
ok I added them